sdk/src/transaction_context.rs

```diff
@@ -15,6 +15,9 @@ use {
     },
 };
 
+/// Maximum permitted size of new allocations per transaction, in bytes
+pub const MAX_PERMITTED_ACCOUNTS_RESIZE_DELTA: i64 = MAX_PERMITTED_DATA_LENGTH as i64 * 4;
```

Comment:
This constant should be documented to explain why this value was chosen. I think that at minimum it should be greater than or equal to `MAX_PERMITTED_DATA_LENGTH`. As for the multiple, might I suggest 2 for now? I think it's reasonable to allow some amount of extra resizing beyond the max but I think we should err on being more conservative for now until something like https://github.com/solana-labs/solana/issues/27029 is implemented